### ISSUE TITLE
suggested modification in ob_blurlock

### ISSUE THREAD

**Issue by ghost** [oberon2007/oblogout-blurlock#1]
```
# !/bin/bash
# /usr/bin/ob_blurlock
# blurlock for oblogout-manjaro

  scrot /tmp/screenshot.png
  convert /tmp/screenshot.png -blur 0x5 /tmp/screenshotblur.png
  i3lock -f -i /tmp/screenshotblur.png

exit 0
```

Don't need a separate `oblogout_blur` this way. Just invoke vanilla oblogout and execute ob_blurlock. Downside is that locking takes a second longer to do the png blur thingy.

~~Also add `scrot` as dependency since we're using it~~

```
Depends On      : i3lock  imagemagick  pygtk  python2-pillow  python2-distutils-extra  python2-dbus
```

**Comment by oberon-manjaro** [comment]
Like that you can use it ONLY with blurlock. Ok. But also oblogout-manjaro has some other features that are not included in vanilla oblogout: It offers the option to switch user and above all it works with openrc!

**Comment by ghost** [comment]
Oh by "vanilla" i didn't mean the upstream `oblogout` package. I was referring to the `oblogout` command (instead of oblogout_blur) from your package. Frankly `oblogout_blur` does nothing but take a screenshot, which i'm doing within `ob_blurlock` itself.

```
$ pacman -Ql oblogout-manjaro |grep bin
oblogout-manjaro /usr/bin/
oblogout-manjaro /usr/bin/fluxboxexit
oblogout-manjaro /usr/bin/ob_blurlock <--------- does the job of scrot-ing instead of oblogout_blur
oblogout-manjaro /usr/bin/oblogout <--------- Use this
oblogout-manjaro /usr/bin/oblogout_blur <---------- Drop this
```

As for switching users, or openRC commands the oblogout.conf file should take care of that depending on the DE. Here's mine for lightdm/lxqt/systemd

```
[commands]
lock = ob_blurlock
logout = lxqt-leave --logout
switch = dm-tool switch-to-greeter
suspend = lxqt-leave --suspend
hibernate = lxqt-leave --hibernate
restart = lxqt-leave --reboot
shutdown = lxqt-leave --shutdown
```

**Comment by ghost** [comment]
In the end all oblogout does is show a few buttons and run the command associated with them. Right ?
So you can set any command in /etc/oblogout.conf as required. For any DM, DE or init.

If somebody does not want the blur feature, might as well use `i3lock` command directly.

**Comment by oberon-manjaro** [comment]
Yes. What's the advantage of losing `oblogout_blur`? Removing it from the package will break systems currently using the command... You are in any case right that depends **scrot** needs to be added ;)

**Comment by oberon-manjaro** [comment]
In fact it should be an optdepends. If you feel like it you can still use it without blurlock... :)

**Comment by ghost** [comment]
Hmm. Break systems it will, and blame you I must :stuck_out_tongue: 
It's kinda elitist that one can use blurlock only through the oblogout interface, whereas we can directly use ob_blurlock on whiskermenu, xautolock (or anything else that can invoke a lock command) with a simple tweak.
Intention was to keep the oblogout and the blurlock components independent so we get to choose which one to use and when. For example i have set my `xautolock` to use blurlock after 30 minutes of inactivity which isn't possible on your setup. Yet I haven't broken oblogout's functionality in any way as far as I can tell.

**Comment by oberon-manjaro** [comment]
I see... I have to meditate on this - somehow I seem to have come up with a rather confusing setup back then - I think at the moment I'm not understanding everything really myself... especially in regards of the old fluxbox construct... :laughing:

**Comment by ghost** [comment]
`scrot` may not be needed as dependency after all. Since we are already having `imagemagick` , might as well use the same for screenshotting. command is `import`
Here's my new ob_blurlock

```
# !/bin/bash
# /usr/bin/ob_blurlock
# blurlock for oblogout-manjaro

  import -window root /tmp/screenshot.png
  convert /tmp/screenshot.png -blur 0x5 /tmp/screenshotblur.png
  i3lock -f -i /tmp/screenshotblur.png

exit 0
```

**Comment by oberon-manjaro** [comment]
How about we just install your script additionally as **/usr/bin/blurlock**. Like that people can call this one if they like and old installs using the existing setup don't need to change anything?

**Comment by oberon-manjaro** [comment]
included in **oblogout-manjaro 20160929** :)
